src/assets/styles/styles/custom/_schedule-style.scss

```diff
@@ -151,6 +151,7 @@
       background-color: $l4;
     }
 
+    margin-left: 4px;
```

Comment:
dodałam te brakujące 4px dla urlopu, kwarantanny i L4, żeby wszystko wyświetlało się w równo w jednej linii

Reply:
Ogólnie 4px to całkiem sporo, ale tak wcześniej były wyrównane pozostałe komórki U, K, L4, więc się dostosowałam. Na Figmie nie jest to określone jednoznacznie:
![pict](https://user-images.githubusercontent.com/48785655/107128730-5d1b1e80-68c0-11eb-9320-93d28195b50d.png)

Reply:
Podziwiam za dokładność. Na Figmie mamy trochę bałagan i stad wynika ten brak konsekwencji. Moim zdaniem jest ładnie, ale pokażemy to Adze jutro na spotkaniu i zapytamy czy to może być zrobione w taki sposób.